lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -33,6 +70,26 @@ <h2 class="message-title hd-5 emphasized">${accomplishment_banner_opening}</h2>
                 <p class="message-copy copy copy-base emphasized">${accomplishment_banner_congrats}</p>
                 <div class="message-actions">
                     <h3 class="sr-only">${_("Take this with you:")}</h3>
```

Comment:
@catong -- if you catch up with @talbs he can probably provide you with a screenshot of the copy on the certificate page -- most of the strings you are commenting on were put in place by the UX team